### ISSUE TITLE
Modman file with js links wrong

### ISSUE THREAD

**Issue by robbanl** [ebizmarts/magemonkey#72]
Hi,

The following line in modman is incorrect and causes 404 errors in frontend

```
js/ebizmarts/abandonedcart                                              js/ebizmarts/abandonedcart
```

It should be

```
js/ebizmarts/*                                             js/ebizmarts/
```

**Comment by robbanl** [comment]
I see it's fixed in commit 594ce10b01420f48641cdfb0927f12ba02b9c587 When will it be merged?

**Comment by Santiagoebizmarts** [comment]
@robbanl Thanks for your feedback. On next release this will be fixed.
Sorry for the troubles caused.
